system/LINK/bin/fix_auth

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# description: ManageIQ appliance console launcher
+#
+
+if [ -d /var/www/miq/vmdb ] ; then
+  cd /var/www/miq/vmdb
+else
+  cd $(dirname $0)/../../../vmdb
```

Comment:
@kbrock I like the `if` leg of the conditional, it makes sense to make it easier since the `fix_auth` shell script will be symlinked in the appliance's `/bin` directory.  I don't know how to use the `else` condition in a dev setup.